compiler/Parser.fir

```diff
@@ -36,15 +37,15 @@ main(args: Array[Str]):
                 panic("Parse error: unexpected end of input")
             else:
                 panic("Parse error: unexpected token `tokens.get(cursor)`")
-        Result.Ok(tree):
-            tree
-
-    # if printAst:
-    #     let buf = StrBuf.withCapacity(1_000_000)
-    #     for topDecl: ParseTree[Token, NonTerminal] in result.tree.asNonTerminal(NonTerminal.Module).iter():
-    #         topDecl.toDoc().renderToBuf(80, buf)
-    #         print(buf.toStr())
-    #         buf.clear()
+        Result.Ok(tree): tree
+
+
+# if printAst:
+#     let buf = StrBuf.withCapacity(1_000_000)
+#     for topDecl: ParseTree[Token, NonTerminal] in result.tree.asNonTerminal(NonTerminal.Module).iter():
+#         topDecl.toDoc().renderToBuf(80, buf)
+#         print(buf.toStr())
+#         buf.clear()
```

Comment:
This looks like a bug.

Reply:
This is related to trailing comment handling. This becomes a part of the next token which is `showUsageAndExit` at column 0. So the comment gets moved to column 0.

It's OK for now.